specification/fluidrelay/resource-manager/Microsoft.FluidRelay/preview/2021-03-12-preview/fluidrelay.json

```diff
@@ -0,0 +1,718 @@
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        }
+      },
+      "description": "The updatable properties of a Fluid Relay server"
+    },
+    "FluidRelayServerList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FluidRelayServer"
+          },
+          "description": "A sequence of FluidRelay servers.",
+          "x-adl-name": "FluidRelayServer[]"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "A link to the next page of results, if any."
+        }
+      },
+      "description": "Paged response",
+      "required": [
+        "value"
+      ],
+      "x-adl-name": "Page<FluidRelayServer>"
+    },
+    "FluidRelayServer": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FluidRelayServerProperties",
+          "description": "The resource-specific properties for this resource.",
+          "readOnly": false,
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System meta data for this resource, including creation and modification information.",
+          "readOnly": true
+        }
+      },
+      "description": "A FluidRelay Server.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "x-ms-azure-resource": true
+    },
+    "FluidRelayServerKeys": {
+      "type": "object",
+      "properties": {
+        "key1": {
+          "type": "string",
+          "format": "password",
+          "description": "The primary key for this server",
+          "readOnly": true
+        },
+        "key2": {
+          "type": "string",
+          "format": "password",
+          "description": "The secondary key for this server",
+          "readOnly": true
+        }
+      },
+      "description": "The set of available keys for this server."
+    },
+    "RegenerateKeyRequest": {
+      "type": "object",
+      "properties": {
+        "keyName": {
+          "type": "string",
+          "enum": [
+            "key1",
+            "key2"
+          ],
+          "x-ms-enum": {
+            "modelAsString": false,
+            "name": "KeyName"
+          },
+          "x-adl-name": "key1 | key2",
+          "description": "The key to regenerate."
+        }
+      },
+      "description": "Specifies which key should be generated.",
+      "required": [
+        "keyName"
+      ]
+    },
+    "FluidRelayServerProperties": {
+      "type": "object",
+      "properties": {
+        "frsTenantId": {
```

Comment:
>frsTenantId [](http://example.com/codeflow?start=9&length=11)

Consider not using the frs abbreviation. ARM guidance is to use the words rather than abbreviations or acronyms.

Reply:
Can we keep the name here? This is our team's common used name, otherwise we will need additional changes for this. Thanks.